src/app/json-editor/json-editor.component.html

```diff
@@ -12,7 +12,7 @@
             <input type="button" title="Minify" value="Minify" class="btn btn-primary btn-block" (click)="minifyJSON()">
 
             <input type="button" title="JSON Viewer" value="JSON Viewer" class="btn btn-primary btn-block"
-                (click)="jsonViewer()" disabled>
+                (click)="jsonViewer()">
```

Comment:
Keep jsonViewer button disabled, as it is not working..

Reply:
Kept as it is.